reader_test.go

```
@@ -57,887 +56,1087 @@ func newXXHash(data []byte) uint64 {
+			bytes.Repeat([]byte("x"), 1000),
+			{},
 		}
-		d, err := ParseDict(raw)
+		for _, src := range inputs {
+			e := NewEncoder()
+			compressed := e.AppendCompress(nil, src)
+
+			// ReadAll
+			r1 := NewReader(bytes.NewReader(compressed), nil)
+			readResult, err := io.ReadAll(r1)
+			r1.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `r1.Close` is not checked

(errcheck)